docs/examples.md

```diff
@@ -142,3 +142,17 @@ Installing to existing venv 'pipx'
     - pipx
 done! ✨ 🌟 ✨
 ```
+
+## `pipx upgrade-shared` examples
+
+One use of the upgrade-shared command is to force a `pip` upgrade.
+
+```shell
+> pipx upgrade-shared
+```
+
+Another use of the upgrade-shared command is to pin (temporarily) `pip` to a specific version.
+
+```shell
+> pipx upgrade-shared --pip-args=pip==24.0
```

Comment:
```suggestion
> pipx upgrade-shared --pip-args="pip == 24.0"
```

I think this improves readability.

Reply:
This fails on windows, c.f. https://github.com/mayeut/pipx/actions/runs/8771336857/job/24068962865

Reply:
It should work with single quotes? Updated the suggestion.

Reply:
It also fails on windows: https://github.com/pypa/pipx/actions/runs/8779377197/job/24087325568